app/models/mail_server_log/delivery_status_serializer.rb

```diff
@@ -0,0 +1,26 @@
+class MailServerLog::DeliveryStatusSerializer
+  def self.load(value)
+    return unless value
+    delivery_status_class.new(value.to_sym)
+  end
+
+  def self.dump(value)
```

Comment:
I think the public methods here could do with a little inline documentation about:

- What type of object they take
- What type of object they return